Version 4.0.16 causes black screen on startup

Original question on SO for reference: https://stackoverflow.com/questions/74091196/gluon-attach-4-0-16-causes-black-screen-on-startup.

Full POM:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>

	<groupId>com.demo</groupId>
	<artifactId>demo-client</artifactId>
	<version>1.0.0-SNAPSHOT</version>

	<name>Demo</name>

	<properties>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>

		<minimum-maven-version>3.8.4</minimum-maven-version>
		<versions-maven-plugin-version>2.12.0</versions-maven-plugin-version>
		<maven-enforcer-plugin-version>3.1.0</maven-enforcer-plugin-version>
		<maven-compiler-plugin-version>3.10.1</maven-compiler-plugin-version>
		<maven-surefire-plugin-version>3.0.0-M6</maven-surefire-plugin-version>
		<javafx-maven-plugin-version>0.0.8</javafx-maven-plugin-version>
		<gluonfx-maven-plugin-version>1.0.16-SNAPSHOT</gluonfx-maven-plugin-version>

		<java-version>17</java-version>
		<javafx-version>19</javafx-version>
		<jackson-version>2.14.0-rc2</jackson-version>
		<charm-version>6.2.2</charm-version>
		<attach-version>4.0.16</attach-version>

		<lombok-version>1.18.24</lombok-version>
		<junit-version>5.7.0</junit-version>

		<main.class>com.demo.MyApplication</main.class>
		<app.identifier>${main.class}</app.identifier>
		<app.description>The Demo app</app.description>
		<package.type />
		<mac.app.store />
		<mac.signing.user.name />
		<bundle.version />
		<bundle.short.version />
		<version.code />
		<provided.keystore.path />
		<provided.keystore.password />
		<provided.key.alias />
		<provided.key.alias.password />
	</properties>

	<dependencies>
		<dependency>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok</artifactId>
			<version>${lombok-version}</version>
			<scope>provided</scope>
		</dependency>

		<dependency>
			<groupId>org.openjfx</groupId>
			<artifactId>javafx-controls</artifactId>
			<version>${javafx-version}</version>
		</dependency>
		<dependency>
			<groupId>org.openjfx</groupId>
			<artifactId>javafx-media</artifactId>
			<version>${javafx-version}</version>
		</dependency>
		<dependency>
			<groupId>com.fasterxml.jackson.core</groupId>
			<artifactId>jackson-databind</artifactId>
			<version>${jackson-version}</version>
		</dependency>

		<dependency>
			<groupId>com.gluonhq</groupId>
			<artifactId>charm-glisten</artifactId>
			<version>${charm-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>display</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>lifecycle</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>statusbar</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>storage</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>position</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>accelerometer</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>magnetometer</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>audio</artifactId>
			<version>${attach-version}</version>
		</dependency>
		<dependency>
			<groupId>com.gluonhq.attach</groupId>
			<artifactId>util</artifactId>
			<version>${attach-version}</version>
		</dependency>
	</dependencies>

	<repositories>
		<repository>
			<id>Gluon</id>
			<url>https://nexus.gluonhq.com/nexus/content/repositories/releases</url>
		</repository>
		<repository>
			<id>snapshot</id>
			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
		</repository>
	</repositories>

	<pluginRepositories>
		<pluginRepository>
			<id>snapshot</id>
			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
		</pluginRepository>
	</pluginRepositories>

	<build>
		<plugins>
			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-enforcer-plugin</artifactId>
				<version>${maven-enforcer-plugin-version}</version>
				<executions>
					<execution>
						<id>enforce-maven</id>
						<goals>
							<goal>enforce</goal>
						</goals>
						<configuration>
							<rules>
								<requireMavenVersion>
									<version>${minimum-maven-version}</version>
								</requireMavenVersion>
							</rules>
						</configuration>
					</execution>
				</executions>
			</plugin>
		
			<plugin>
				<groupId>org.codehaus.mojo</groupId>
				<artifactId>versions-maven-plugin</artifactId>
				<version>${versions-maven-plugin-version}</version>
			</plugin>

			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-compiler-plugin</artifactId>
				<version>${maven-compiler-plugin-version}</version>
				<configuration>
					<release>${java-version}</release>
 					<compilerArgs>
						<!-- <arg>add a  - -enable-preview</arg> -->
					</compilerArgs>
				</configuration>
			</plugin>

			<plugin>
				<groupId>org.openjfx</groupId>
				<artifactId>javafx-maven-plugin</artifactId>
				<version>${javafx-maven-plugin-version}</version>
				<configuration>
					<mainClass>${main.class}</mainClass>
				</configuration>
			</plugin>

			<plugin>
				<groupId>com.gluonhq</groupId>
				<artifactId>gluonfx-maven-plugin</artifactId>
				<version>${gluonfx-maven-plugin-version}</version>
				<configuration>
					<verbose>true</verbose>
					<javafxStaticSdkVersion>${javafx-version}</javafxStaticSdkVersion>
					<target>${gluonfx.target}</target>
					<attachList>
						<list>display</list>
						<list>lifecycle</list>
						<list>statusbar</list>
						<list>storage</list>
						<list>position</list>
						<list>accelerometer</list>
						<list>magnetometer</list>
						<list>audio</list>
					</attachList>
					<resourcesList>
						<item>.*/LongBeep.wav$</item>
						<item>.*/ShortBeep.wav$</item>
					</resourcesList>
					<mainClass>${main.class}</mainClass>
					<appIdentifier>${app.identifier}</appIdentifier>
					<releaseConfiguration>
						<vendor>Gluon</vendor>
						<description>${app.description}</description>
						<packageType>${package.type}</packageType>
						<!-- for macOS/iOS -->
						<macAppStore>${mac.app.store}</macAppStore>
						<bundleShortVersion>${bundle.short.version}</bundleShortVersion>
						<bundleVersion>${bundle.version}</bundleVersion>
						<!-- for Android -->
						<versionCode>${version.code}</versionCode>
						<providedKeyStorePath>${provided.keystore.path}</providedKeyStorePath>
						<providedKeyStorePassword>${provided.keystore.password}</providedKeyStorePassword>
						<providedKeyAlias>${provided.key.alias}</providedKeyAlias>
						<providedKeyAliasPassword>${provided.key.alias.password}</providedKeyAliasPassword>
					</releaseConfiguration>
				</configuration>
			</plugin>
		</plugins>
	</build>

	<profiles>
		<profile>
			<id>desktop</id>
			<activation>
				<activeByDefault>true</activeByDefault>
			</activation>
			<properties>
				<gluonfx.target>host</gluonfx.target>
			</properties>
		</profile>
		<profile>
			<id>mac</id>
			<properties>
				<package.type>pkg</package.type>
				<mac.app.store>false</mac.app.store>
				<bundle.version>${env.GITHUB_RUN_NUMBER}</bundle.version>
				<bundle.short.version>1.0</bundle.short.version>
			</properties>
		</profile>
		<profile>
			<id>macstore</id>
			<properties>
				<package.type>pkg</package.type>
				<mac.app.store>true</mac.app.store>
				<bundle.version>1.${env.GITHUB_RUN_NUMBER}</bundle.version>
				<bundle.short.version>1.6</bundle.short.version>
			</properties>
		</profile>
		<profile>
			<id>ios</id>
			<properties>
				<gluonfx.target>ios</gluonfx.target>
				<bundle.version>${env.GITHUB_RUN_NUMBER}</bundle.version>
				<bundle.short.version>1.0</bundle.short.version>
			</properties>
		</profile>
		<profile>
			<id>android</id>
			<properties>
				<gluonfx.target>android</gluonfx.target>
				<app.identifier>com.my.demo</app.identifier>
				<version.code>${env.GITHUB_RUN_NUMBER}</version.code>
				<provided.keystore.path>${env.GLUON_ANDROID_KEYSTOREPATH}</provided.keystore.path>
				<provided.keystore.password>${env.GLUON_ANDROID_KEYSTORE_PASSWORD}</provided.keystore.password>
				<provided.key.alias>${env.GLUON_ANDROID_KEYALIAS}</provided.key.alias>
				<provided.key.alias.password>${env.GLUON_ANDROID_KEYALIAS_PASSWORD}</provided.key.alias.password>
			</properties>
		</profile>
		<profile>
			<id>pi</id>
			<properties>
				<gluonfx.target>linux-aarch64</gluonfx.target>
			</properties>
		</profile>
	</profiles>
</project>
```

Using graalvm-svm-java17-linux-gluon-22.1.0.1-Final.

Causes the following error from adb logcat:

```
10-17 15:04:24.966 26884 26908 D GluonAttach: Util :: Load className com/gluonhq/helloandroid/DalvikDisplayService
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504] JNI DETECTED ERROR IN APPLICATION: JNI NewStringUTF called with pending exception java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at void android.os.Handler.<init>(android.os.Handler$Callback, boolean) (Handler.java:203)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at void android.os.Handler.<init>() (Handler.java:117)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at void android.content.ClipboardManager$2.<init>(android.content.ClipboardManager) (ClipboardManager.java:84)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at void android.content.ClipboardManager.<init>(android.content.Context, android.os.Handler) (ClipboardManager.java:84)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at android.content.ClipboardManager android.app.SystemServiceRegistry$12.createService(android.app.ContextImpl) (SystemServiceRegistry.java:379)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.app.SystemServiceRegistry$12.createService(android.app.ContextImpl) (SystemServiceRegistry.java:377)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.app.SystemServiceRegistry$CachedServiceFetcher.getService(android.app.ContextImpl) (SystemServiceRegistry.java:1402)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.app.SystemServiceRegistry.getSystemService(android.app.ContextImpl, java.lang.String) (SystemServiceRegistry.java:1354)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.app.ContextImpl.getSystemService(java.lang.String) (ContextImpl.java:1695)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.view.ContextThemeWrapper.getSystemService(java.lang.String) (ContextThemeWrapper.java:171)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at java.lang.Object android.app.Activity.getSystemService(java.lang.String) (Activity.java:6008)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   at void com.gluonhq.helloandroid.Util.<init>(android.app.Activity) (Util.java:53)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]     in call to NewStringUTF
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504] "Thread-6" daemon prio=5 tid=16 Runnable
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   | group="main" sCount=0 dsCount=0 flags=0 obj=0x12e80000 self=0x7e74f27800
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   | sysTid=26908 nice=0 cgrp=default sched=0/0 handle=0x7e604ff4f0
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   | state=R schedstat=( 106877037 2373803 92 ) utm=9 stm=1 core=2 HZ=100
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   | stack=0x7e60405000-0x7e60407000 stackSize=1005KB
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   | held mutexes= "mutator lock"(shared held)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #00 pc 00000000003973ec  /system/lib64/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_9ArtMethodEPv+212)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #01 pc 000000000045dba0  /system/lib64/libart.so (_ZNK3art6Thread9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMapb+348)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #02 pc 00000000002d57b8  /system/lib64/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1040)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #03 pc 00000000002d5c38  /system/lib64/libart.so (_ZN3art9JavaVMExt9JniAbortVEPKcS2_St9__va_list+116)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #04 pc 0000000000108878  /system/lib64/libart.so (_ZN3art11ScopedCheck6AbortFEPKcz+148)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #05 pc 00000000001083a4  /system/lib64/libart.so (_ZN3art11ScopedCheck11CheckThreadEP7_JNIEnv+536)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #06 pc 0000000000106840  /system/lib64/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE+856)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #07 pc 00000000000fd004  /system/lib64/libart.so (_ZN3art8CheckJNI12NewStringUTFEP7_JNIEnvPKc+648)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #08 pc 00000000019c07f0  /data/app/com.gps.demo-vSN-YTEtwQ1kgcyL1xFIMg==/lib/arm64/libsubstrate.so (registerDalvikClass+448)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #09 pc 00000000019be2e4  /data/app/com.gps.demo-vSN-YTEtwQ1kgcyL1xFIMg==/lib/arm64/libsubstrate.so (initializeDisplayDalvikHandles+36)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #10 pc 00000000019bde90  /data/app/com.gps.demo-vSN-YTEtwQ1kgcyL1xFIMg==/lib/arm64/libsubstrate.so (JNI_OnLoad_display+140)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   native: #11 pc 00000000002e2ea8  /data/app/com.gps.demo-vSN-YTEtwQ1kgcyL1xFIMg==/lib/arm64/libsubstrate.so (JNILibraryInitializer_callOnLoadFunction_7833a452cbe2b969eec248ff45fd5fe129f3cbdc+104)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]   (no managed stack frames)
10-17 15:04:24.982 26884 26908 F zygote64: java_vm_ext.cc:504]
```

Happens with GluonFX 1.0.15 as well. Attach 4.0.15 works fine.

Device details:
Samsung Galaxy S7
![Screenshot_20221017-163019_Settings](https://user-images.githubusercontent.com/37422899/196190281-07551ca0-8d00-47a2-9a9c-2f9ff2063f51.jpg)

I deleted some caches and restarted the gradle daemon. Now I'm getting another error during `:accelerometer:androidBuild`:
```
Could not write standard input to Gradle build daemon.
java.io.IOException: Stream closed
        at java.base/java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:445)
        at java.base/java.io.OutputStream.write(OutputStream.java:162)
        at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
        at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
        at org.gradle.process.internal.streams.ExecOutputHandleRunner.forwardContent(ExecOutputHandleRunner.java:68)
        at org.gradle.process.internal.streams.ExecOutputHandleRunner.run(ExecOutputHandleRunner.java:53)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

FAILURE: Build failed with an exception.

* What went wrong:
Unable to start the daemon process.
This problem might be caused by incorrect configuration of the daemon.
For example, an unrecognized jvm option is used.
Please refer to the User Manual chapter on the daemon at https://docs.gradle.org/7.5/userguide/gradle_daemon.html
Process command line: /usr/lib/jvm/java-17-openjdk-amd64/bin/java -XX:MaxPermSize=1024m -XX:MaxMetaspaceSize=1g --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.lang.invoke=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.prefs/java.util.prefs=ALL-UNNAMED --add-opens java.prefs/java.util.prefs=ALL-UNNAMED --add-opens java.base/java.nio.charset=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.util.concurrent.atomic=ALL-UNNAMED -Xms512M -Xmx4g -Dfile.encoding=UTF-8 -Duser.country -Duser.language=en -Duser.variant -cp /home/nlisker/.gradle/wrapper/dists/gradle-7.5-bin/7u84gh846ok982rnccbf0jshgv/gradle-7.5/lib/gradle-launcher-7.5.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 7.5
Please read the following process output to find out more:
-----------------------
Unrecognized VM option 'MaxPermSize=1024m'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```
Indeed, in Java 17 that option was [removed](https://docs.oracle.com/en/java/javase/17/docs/specs/man/java.html#removed-java-options). Any idea where it comes from?

The error with the non-final variable repeats after I clear the caches and builds. If I try to build again then it somehow succeeds. However, I noticed that when building the application, it takes the 4.0.19-SNAPSHOT build from the online repository, which is why the JNI error repeated itself. I tried to change the gradle.properties version to 4.0.20-SNAPSHOT to make sure it doesn't come from a remote repo. Then I built the client again and got this exception:

```
05-03 18:53:30.230 18526 18546 D GraalCompiled: Exception in Application start method
05-03 18:53:30.231 18526 18546 D GraalCompiled: QuantumRenderer: shutdown
05-03 18:53:30.231 18526 18546 D GraalCompiled: Exception in thread "main" java.lang.RuntimeException: Exception in Application start method
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:893)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at java.lang.Thread.run(Thread.java:833)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:704)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.oracle.svm.core.posix.thread.PosixPlatformThreads.pthreadStartRoutine(PosixPlatformThreads.java:202)
05-03 18:53:30.233 18526 18546 D GraalCompiled: Caused by: java.lang.UnsatisfiedLinkError: no util in java.library.path
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.oracle.svm.core.jdk.NativeLibrarySupport.loadLibraryRelative(NativeLibrarySupport.java:132)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at java.lang.ClassLoader.loadLibrary(ClassLoader.java:47)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at java.lang.Runtime.loadLibrary0(Runtime.java:818)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at java.lang.System.loadLibrary(System.java:1989)
05-03 18:53:30.233 18526 18546 D GraalCompiled:         at com.gluonhq.attach.util.Platform.<clinit>(Platform.java:83)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.attach.util.impl.DefaultServiceFactory.getInstance(DefaultServiceFactory.java:56)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.attach.util.Services.get(Services.java:85)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.attach.display.DisplayService.create(DisplayService.java:95)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.impl.charm.glisten.util.DeviceSettings.hasNotch(DeviceSettings.java:27)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.impl.charm.glisten.util.DeviceSettings.setup(DeviceSettings.java:45)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.impl.charm.glisten.util.DeviceSettings.dedicatedSetup(DeviceSettings.java:38)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.charm.glisten.control.AppBar.<init>(AppBar.java:127)
05-03 18:53:30.234 18526 18546 D GraalCompiled:         at com.gluonhq.charm.glisten.application.AppManager.start(AppManager.java:258)
```
So I suspect there really is a problem in building Util.

--- a/modules/util/src/main/native/android/dalvik/Util.java
+++ b/modules/util/src/main/native/android/dalvik/Util.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2022, Gluon
+ * Copyright (c) 2020, 2023, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by

--- a/modules/util/src/main/native/android/dalvik/Util.java
+++ b/modules/util/src/main/native/android/dalvik/Util.java
@@ -50,8 +50,17 @@ public class Util {
         Util.activity = activity;
         Log.v(TAG, "Util <init>");
         if (activity != null) {
-            Util.clipboard = (ClipboardManager) activity.getSystemService(Context.CLIPBOARD_SERVICE);
-            syncClipboardFromOS();
+            new Handler(activity.getMainLooper()).postDelayed(new Runnable() {
+
+                @Override
+                public void run() {
+                    if (debug) {
+                        Log.v(TAG, "Util init clipboard");
+                    }
+                    Util.clipboard = (ClipboardManager) activity.getSystemService(Context.CLIPBOARD_SERVICE);
+                    syncClipboardFromOS();
+                }
+            }, 0);
         }
     }
 

--- a/modules/util/src/main/native/android/dalvik/Util.java
+++ b/modules/util/src/main/native/android/dalvik/Util.java
@@ -50,14 +50,14 @@ public class Util {
         Util.activity = activity;
         Log.v(TAG, "Util <init>");
         if (activity != null) {
-            new Handler(activity.getMainLooper()).postDelayed(new Runnable() {
+            new Handler(Util.activity.getMainLooper()).postDelayed(new Runnable() {
 
                 @Override
                 public void run() {
                     if (debug) {
                         Log.v(TAG, "Util init clipboard");
                     }
-                    Util.clipboard = (ClipboardManager) activity.getSystemService(Context.CLIPBOARD_SERVICE);
+                    Util.clipboard = (ClipboardManager) Util.activity.getSystemService(Context.CLIPBOARD_SERVICE);
                     syncClipboardFromOS();
                 }
             }, 0);

